contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -492,7 +492,7 @@ SELECT
 CAST(name AS SYS.SYSNAME) AS name,
 CAST(
   CASE name
-    WHEN 'public' THEN 0
+    WHEN 'public' THEN 1
```

Comment:
Please incorporate this change for every object where public role has been used like user_token and any other possible objects.